### ISSUE TITLE
Дизайн

### ISSUE THREAD

**Issue by nikolai-shabalin** [nikolai-shabalin/css-properties#6]
Найти пример простого дизайна или задизайнить самому

**Comment by Gcusok** [comment]
Правильно ли я понимаю, что нужно кидать что-то типо референсов допустим сюда в комментарии и по средствам заимствований и тд, будет приниматься решение ?

**Comment by nikolai-shabalin** [comment]
Да, можно таким образом поступить. Сейчас проект это голый HTML с парой строк стилей. 
Поэтому позаимствовать хорошие примеры отличный 1 шаг.
